package.json

```diff
@@ -108,14 +108,17 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
-    "npm-run-all": "4.1.5",
+    "npm-run-all": "^4.1.5",
     "prop-types": "^15.7.2",
     "stylelint": "^16.1.0",
     "stylelint-config-recommended-scss": "^14.0.0",
     "stylelint-scss": "^6.0.0",
     "ts-patch": "^3.1.1",
     "typescript": "^5.3.3"
   },
+  "overrides": {
+    "cross-spawn": "^7.0.5"
+  },
```

Comment:
You can avoid using overrides (and I would also suggest it) by running npm audit fix. This will update the dependencies in your package-lock

Reply:
AFAIK `npm audit fix` does not do overrides and this CVE cannot be fixed without overrides. I tried running it locally and it still stayed at the vulnerable version.